external_search.py

```diff
@@ -326,6 +331,7 @@ def setup_index(self, new_index=None, **index_settings):
         """
         index_name = new_index or self.works_index
         if self.indices.exists(index_name):
+            self.log.info("Deleting index %s", index_name)
```

Comment:
This way someone who runs bin/repair/search_index hears about the index being deleted.